pydicom/tests/test_filewriter.py

```diff
@@ -1521,9 +1521,9 @@ def test_private_tag_vr_from_implicit_data(self):
         ds_expl = dcmread(fp)
 
         assert ds_expl[(0x0009, 0x0010)].VR == 'LO'  # private creator
-        assert ds_expl[(0x0009, 0x1001)].VR == 'UN'  # originally LO
-        assert ds_expl[(0x0009, 0x10e7)].VR == 'UN'  # originally UL
-        assert ds_expl[(0x0043, 0x1010)].VR == 'UN'  # originally US
+        assert ds_expl[(0x0009, 0x1001)].VR == 'LO'
+        assert ds_expl[(0x0009, 0x10e7)].VR == 'UL'
+        assert ds_expl[(0x0043, 0x1010)].VR == 'US'
```

Comment:
I wanted to add some unknown private tags here, but apparantly all private tags in our sample data are known.